### PR TITLE
docs: initial POC for publications page

### PR DESCRIPTION
**Pull request by maxandersen**
Will be on http://localhost:4000/publications when you run jekyll.

While learning about Quarkus I browsed randomly online to see what else was out there and
thought it would be cool if we actually had a page to gather external publications related to Quarkus.

This is my first go on taking links I visited recently and see how it would look in a table.

After having done it I think better approach would be to create _publications/* folder like _posts so we
can sort/redner it as we want rather than a massive big CSV file.

Could also be used for a monthly or even weekly "Quarkus News" style blog post linking the latest entries.

WDYT?

** If you are updating a guide, please submit your pull request to the main repository: https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc **

### CODE REVIEW

**Comment by maxandersen** [comment]
screenshot for the lazy ones ;)

**Comment by maxandersen** [comment]
okey - redid this using jekyll-scholar as I was getting crazy editing the tables by hand.

 * use `jekyll-scholar` to have a dynamic way of rendering and querying
   the publications.
 * `/publications` page with ALL publications
 * Example of a "Quarkus News" blog we can make every week/monthh etc.
   with an intro text and then via `bibquery` attribute select which
   publications to refer to. i.e.
   `bibquery:  "@*[date >= 2019-10-01 && date < 2019-11-01]"
   would return all publications published in november.
   Query can use all of the possible fields and combine them.
 * `_layouts/bibliography.html` controls how each publication are rendered.
 * `_bibliography/publications.bib` is the bibtex formatted listed of publications.
   It is a standard format and with jekyll-scholar I could get this created
   much faster than if I were going to implement all the querying/grouping/sorting
   myself.

Known limitation is that group/sorter is global for both the `/publications` page and
blog entries. I think we can live with that for now and I've opened github.com/inukshuk/jekyll-scholar/issues/279
to give us more freedom. In worst case we convert the .bib format to "raw" jekyll data blog
and do our own sorting/grouping.

File: _data/authors.yaml [hunk 1]
```diff
@@ -130,4 +130,11 @@ dakwon:
   emailhash: "ddbf60c0e87ce6ad86f98aaf4fa55ff0"
   job_title: "Software Engineering Intern"
   twitter: ""
-  bio: "Intern at Red Hat working on developer tools."
\ No newline at end of file
+  bio: "Intern at Red Hat working on developer tools."
```
Comment (emmanuelbernard) [inline _data/authors.yaml]: ROTF

**Comment by emmanuelbernard** [comment]
Forgot to ask. Is that sorted in decreasing order of dates (most recent
first)?

**Comment by maxandersen** [comment]
> Looks good to me. Can I publish @maxandersen ?

I would like to do another round of grabbing articles and ask team if they have some to share then post the blog around the 1st. 

> Forgot to ask. Is that sorted in decreasing order of dates (most recent first)?

yes, specific order of types first and then within each category its newest first by publish date.

**Comment by maxandersen** [comment]
actually, let me disable the blog - then we can roll this out quietly, get additional articles added and then post first quarkus news after that.

**Comment by maxandersen** [comment]
removed blog - awaiting travis to verify and we can push it live then. @emmanuelbernard ack?

**Comment by emmanuelbernard** [comment]
@maxandersen can you rebase, GitHub claims conflicts

**Comment by maxandersen** [comment]
i rebased and pushed earlier - its live without blog.

**Pull request closed without merging**